test/test_cpp20/custom_error.cpp

```diff
@@ -7,11 +7,10 @@ namespace
 {
     static bool s_loggerCalled = false;
 
-    // Note that we are checking that the source line number matches expectations.  If lines above this are changed
-    // then this value needs to change as well.
     void FailOnLine15()
     {
         // Validate that handler translated exception
+#line 15 // Force next line to be reported as line number 15
```

Comment:
How was this flaky?  Line 15 should be line 15 unless people are changing this code.

Reply:
It's not flaky, but it's fragile.

Reply:
Ah, that makes sense.

Reply:
The question is it flaky or is it flakey. 🤔